src/routes/api/assistant/+server.ts

```diff
@@ -0,0 +1,110 @@
+import type { RequestHandler } from './$types';
+import { env } from '$env/dynamic/private';
+import { AssistantResponse } from 'ai';
+import OpenAI from 'openai';
+
+const openai = new OpenAI({
+  apiKey: env.GROQ_API_KEY || '',
+});
+
+const homeTemperatures = {
```

Comment:
what is this, please cleanup the code

Reply:
sure, i'll make sure to clean this in next commit.